mq/plugins/vulnerability.py

```diff
@@ -77,7 +77,7 @@ def calculate_id_v2(self, message):
         return hashlib.md5(s).hexdigest()
 
     def onMessage(self, message, metadata):
-        if metadata['doc_type'] != 'vulnerability':
+        if message['type'] != 'vulnerability':
```

Comment:
Need to communicate this change upstream to @arcrose prior to moving to ES 6 so that the changes can be reflected in the vulnerability code that sends events to MozDef to remove doc_type from the event and add type.